db/migrate/20240219121949_create_needed_item.rb

```diff
@@ -0,0 +1,9 @@
+class CreateNeededItem < ActiveRecord::Migration[7.0]
+  def change
+    create_table :needed_items do |t|
```

Comment:
Can we make this `child_needed_items` to make it explicit what this is for?

Reply:
yes we can. My thought was that maybe it will be used on other models also. That's why naming is more abstract

Reply:
addressed
```ruby
class CreateJoinTableChildItem < ActiveRecord::Migration[7.0]
  def change
    create_table :children_items do |t|
      t.references :child, null: false, foreign_key: true
      t.references :item, null: false, foreign_key: true

      t.timestamps
    end

    add_index :children_items, [:child_id, :item_id], unique: true
  end
end
```